docs/README.md

```diff
@@ -10,22 +10,33 @@ The plugin comes with multiple builders able to create Parallels machines,
 depending on the strategy you want to use to build the image. Packer supports
 the following Parallels builders:
 
-- [parallels-iso](/docs/builders/parallels-iso.mdx) - Starts from an ISO
```

Comment:
Any reason why we changed this? I think their internal builder will add the prefix to the files